Jenkinsfile

```diff
@@ -7,7 +7,7 @@ pipeline {
     }
 
     environment {
-        repository = "tylerorg"
+        repository = "index.docker.io/tylerorg"
```

Comment:
my PR and yours will clash here so whoever merges last will need to rebase changes in their branch before merging.

Reply:
I'll change it